alerting/src/main/kotlin/org/opensearch/alerting/BucketLevelMonitorRunner.kt

```diff
@@ -220,6 +224,7 @@ object BucketLevelMonitorRunner : MonitorRunner() {
                     ?.addAll(monitorCtx.alertService!!.convertToCompletedAlerts(keysToAlertsMap))
         }
 
```

Comment:
What are the keys of this map variable and the inner map ?
Can we add some code comments to elaborate about how the docs are held in memory

Reply:
Good callout. I've added a comment to explain that the `alertSampleDocs` map is `Map<TriggerId, Map<BucketKeysHash, List<Alert>>>`.